### PR TITLE
fix: colors type

### PR DESCRIPTION
**Pull request by Arthurmqz**
# Description

Fixed the typology of colors and the  autocomplete 

Fixes # (issue))

## Type of change

Colors Type

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other:

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/react/src/tokens/themes/theme.constants.ts [hunk 1]
```diff
@@ -8,6 +8,56 @@ export enum FontWeight {
...
+  BLUE = 'blue',
+  RED = 'red',
+  BEIGE = 'beige',
+  YELLOW = 'yellow',
+  PURPLE = 'purple',
+  PINK = 'pink',
+  GREEN = 'green',
+  ORANGE = 'orange',
+}
+export enum ExtendedShadeColors {
```
Comment (joaoprcosta) [inline packages/react/src/tokens/themes/theme.constants.ts:49]: What is the difference between `NeutralShadeColors ` and  `ExtendedShadeColors `?

Comment (Arthurmqz) [reply packages/react/src/tokens/themes/theme.constants.ts:49]: Good point, there is no difference. I'll change it to ShadeColors

**Review by vzdiogo — APPROVED** [review]

**Review by joaoprcosta — APPROVED** [review]

File: packages/components/button/src/Button.tsx [hunk 2]
```diff
@@ -1,21 +1,21 @@
 import type { ButtonProps } from './button.types';
 import { ButtonStyled } from './button.styles';
+import theme from '../../../react/src/tokens/themes/theme';
 
 /**
  * Primary UI component for user interaction
  */
 const Button = ({
   primary = false,
   size = 'small',
```
Comment (jessywork) [inline packages/components/button/src/Button.tsx:10]: if you remove the backgroundColor, the user cannot choose the button color

Comment (Arthurmqz) [reply packages/components/button/src/Button.tsx:10]: I agree with you, however, this button component have to be rebuilt, I have only use this color approach to test the type implementation.

**Review by jessywork — APPROVED** [review]

**Pull request merged**